packages/shared/src/mock/metadata.ts

```diff
@@ -151,7 +151,7 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
           signal: makeDataset('twoD_complex', complexType, [2, 2], {
             valueId: 'twoD_cplx',
           }),
-          axes: { position: makeDataset('position', intType, [2]) },
+          axes: { position: makeDataset('position', intType, [3]) },
```

Comment:
I'm adding a value to the mock abscissa array of `/nexus_entry/complex` to test this use case in both `NX Heatmap` and `NX Line`. As expected, there is no visual change (except to the `/nexus_entry/complex/position` dataset itself, of course).